nodebalancer/cmd/nodebalancer/routes.go

```diff
@@ -123,3 +124,34 @@ func lbJSONRPCHandler(w http.ResponseWriter, r *http.Request, blockchain string,
 		return
 	}
 }
+
+// balancesRoute handles the /balances endpoint
+func balancesRoute(w http.ResponseWriter, r *http.Request) {
+	// Get address from query params
+	address := r.URL.Query().Get("address")
```

Comment:
if address is empty `""` I think we need to return everything (but I think it will exceed response limit) or raise an error

Reply:
all balances for all addresses, or maybe this is bad case, and better to raise an error if address not provided

Reply:
That are impossible todo even for quicknode. )